spec/lib/data_update_scripts/add_work_profile_field_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+require Rails.root.join(
+  "lib/data_update_scripts/20210630034523_add_work_profile_field.rb",
+)
+
+describe DataUpdateScripts::AddWorkProfileField do
+  it "adds a new profile field" do
+    expect { described_class.new.run }.to change(ProfileField, :count).by(1)
```

Comment:
I see a lot of our tests use `described_class.new`. Is there any reason we don't use `subject` instead? Barring `subject`'s memoization, they appear to be functionally equivalent.

```ruby
require 'bundler/inline'

gemfile true do
  source 'https://rubygems.org'
  gem 'rspec'
end

require 'rspec/autorun'

RSpec.describe String do
  it 'idk lol' do
    expect(subject).to eq described_class.new
  end
end

# .
#
# Finished in 0.00214 seconds (files took 0.05209 seconds to load)
# 1 example, 0 failures
```

Reply:
The original choice predates me, so I guess I do it mostly for consistency's sake. I wouldn't be surprised if has to do with the `NamedSubject` cop.

Reply:
I believe I may have chosen that early on because it was an RSpec suggestion.